addons/addon-base-raas/packages/base-raas-cfn-templates/src/templates/service-catalog/ec2-rstudio-instance.cfn.yml

```diff
@@ -159,18 +159,10 @@ Resources:
             FromPort: -1
             ToPort: -1
             CidrIp: !Ref AccessFromCIDRBlock
-        - !If
-          - AppStreamEnabled
-          - DestinationSecurityGroupId:
-              Fn::ImportValue: !Sub "${SolutionNamespace}-CfnEndpointSecurityGroup"
-            IpProtocol: '-1'
-          - !Ref "AWS::NoValue"
       SecurityGroupIngress:
         - !If
           - AppStreamEnabled
-          - SourceSecurityGroupId:
-              Fn::ImportValue: !Sub "${SolutionNamespace}-SwbAppStreamSG"
-            IpProtocol: '-1'
+          - !Ref "AWS::NoValue"
```

Comment:
I believe these are placeholder changes for RStudio, and we'll need to test AppStream with the ALB implementation eventually, right?